pkg/splunk/enterprise/afwscheduler.go

```diff
@@ -735,27 +794,62 @@ installPhase:
 
 		default:
 			for _, installWorker := range pplnPhase.q {
+				if enterpriseApi.ScopeLocal != getAppSrcScope(installWorker.afwConfig, installWorker.appSrcName) {
+					scopedLog.Error(nil, "Install worker with wrong scope", "name", installWorker.cr.GetName(), "namespace", installWorker.cr.GetNamespace(), "pod name", installWorker.targetPodName, "App name", installWorker.appDeployInfo.AppName, "digest", installWorker.appDeployInfo.ObjectHash)
```

Comment:
nit: "wrong" is probably a strong word to use here, maybe something like - "Install worker with scope other than local scope" or something else?